### PR TITLE
S2 us3 view story

### PR DESCRIPTION
**Pull request by Vannou28**

### CODE REVIEW

**Review by RaphaelBS-WCS — DISMISSED** [review]
Approved

File: src/View/History/index.html.twig [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+{% block content %}
+    <section class="fullStory" id="story">
+        <h2>Qui suis-je?</h2>
+        <div class="container">
+            <h3>Mon parcours</h3>
+            <div class="row d-flex align-items-start justify-content-around p-5">
+                <h3>A 8ans :</h3>
+                <div class="col-12 col-sm-12 col-md-3 col-lg-3">
+                    <img src="../assets/images/history/8ans.webp" class="img-fluid" alt="Responsive image">
```
Comment (LucyMerlier) [inline src/View/History/index.html.twig:15]: alt="Responsive image" => not a valid alt

File: src/View/History/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                        je leur demandais de partir et de se taire en mettant mes mains sur mes oreilles...</br>
+                        Je ne saurais dire combien de semaines ou de mois cela à duré, car pour moi c'était comme si le temps 
+                        s'était arrêté et que chaque jour, chaque nuit était identique...
+                    </p>
+                </div>
+            </div>
+            <div class="row d-flex align-items-start justify-content-around p-5">
+                <h3>De 9 à 20 ans :</h3>
+                <div class="col-12 col-sm-12 col-md-3 col-lg-3">
+                    <img src="../assets/images/history/9-20ans.webp" class="img-fluid" alt="Responsive image">
```
Comment (LucyMerlier) [inline src/View/History/index.html.twig:34]: same as above

File: src/View/History/index.html.twig [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                        je sentais que quelque chose n'était pas "normal" et ma "différence" s'est confirmée en grandissant.</br>
+                        Mon père à tenté tant bien que mal de me fermer ce qu'il appelait le 3ème oeil mais cela à continué 
+                        chroniquement à l'adolescence jusqu'à ce que je rentre dans la vie active à l'âge de 20 ans.
+                    </p>
+                </div>
+            </div>
+            <div class="row d-flex align-items-start justify-content-around p-5">
+                <h3>A 30 ans :</h3>
+                <div class="col-12 col-sm-12 col-md-3 col-lg-3">
+                    <img src="../assets/images/history/30ans.webp" class="img-fluid" alt="Responsive image">
```
Comment (LucyMerlier) [inline src/View/History/index.html.twig:47]: same as above

File: src/View/History/index.html.twig [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+                        alors j'ai acheté mon premier jeu de cartes et Stéphane est venu 3 semaines en France, il a été mon premier véritable "cobaye"... 
+                        Je n'arrive toujours pas à croire que cela s'est produit... comme si tout s'était débloqué avec lui, j'ai eu des visions, des mots qu'on me disait 
+                        et qu'enfin je pouvais "capter", je me suis sentie bien , heureuse, sereine, comme si c'était enfin mon chemin de vie.</br>
+                    </p>
+                </div>
+            </div>
+            <div class="row d-flex align-items-start justify-content-around p-5">
+                <h3>A 42 ans :</h3>
+                <div class="col-12 col-sm-12 col-md-3 col-lg-3">
+                    <img src="../assets/images/history/42ans.jpg" class="img-fluid" alt="Responsive image">
```
Comment (LucyMerlier) [inline src/View/History/index.html.twig:66]: same as above

File: src/View/History/index.html.twig [hunk 5]
```diff
@@ -0,0 +1,76 @@
+{% extends 'layout.html.twig' %}
+
+{% block stylesheet %}
+    <link rel="stylesheet" href="/assets/css/history/indexHistory.css">
+{% endblock %}
+
+{% block content %}
+    <section class="fullStory" id="story">
```
Comment (LucyMerlier) [inline src/View/History/index.html.twig:8]: use kebab-case for classnames

File: src/Controller/HistoryController.php [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+     * @return string
+     * @throws \Twig\Error\LoaderError
+     * @throws \Twig\Error\RuntimeError
+     * @throws \Twig\Error\SyntaxError
+     */
+    public function index()
+    {
+        return $this->twig->render('History/index.html.twig');
+    }
+}
```
Comment (LucyMerlier) [inline src/Controller/HistoryController.php:26]: is it really necessary to have a dedicated controller just for this page?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: src/Controller/HistoryController.php (hunk 6 above)
Comment (Vannou28) [reply src/Controller/HistoryController.php:26]: I will modify it this weekend if you see at the level of the controller

Comment (Vannou28) [reply src/Controller/HistoryController.php]: the page is transferred to the HomeControler the link in the index story is changed and transferred. vannou28

**Review by RaphaelBS-WCS — APPROVED** [review]
Approved

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**